src/plasmapy/diagnostics/charged_particle_radiography/synthetic_radiography.py

```diff
@@ -661,6 +670,11 @@ def create_particles(
             on the image, but will well-sample the field grid with a
             smaller number of particles. The default is ``'monte-carlo'``.
 
+        source_vdir : `numpy.ndarray`, shape (3), optional
```

Comment:
```suggestion
        source_vdir : (3,) |array_like|, default: (0, 0, 1)  # please correct default!
```
I've been trying to be consistent in the way that we describe [type specifications](https://docs.plasmapy.org/en/stable/contributing/doc_guide.html#docstring-guidelines) in docstrings.  Alas, the numpydoc standard hasn't defined this yet (see https://github.com/numpy/numpydoc/pull/423).

I also didn't double check that this would be the right default for this.

Reply:
Sure, I can change to |array_like|. The default is not constant (defaults to the source-detector axis),  so not sure what to write for that? I suppose literally the default is `None`.

Reply:
In that case, `None` will work best!

Reply:
Sure, done!